oauth2client/contrib/django_util/signals.py

```diff
@@ -20,7 +20,7 @@
 
 import django.dispatch
 
-"""Signal that fires when  OAuth2 Flow has completed.
+""" Signal that fires when  OAuth2 Flow has completed.
```

Comment:
There are two spaces between the words "when" and "OAuth2". No need to fix in this already-large enough pull request.